algorithms/linfa-tsne/examples/mnist_plot.plt

```diff
@@ -1,7 +1,2 @@
-set style increment user
-set style line 1 lc rgb 'red'
-set style line 2 lc rgb 'blue'
-set style line 3 lc rgb 'green'
```

Comment:
Can you include these 4 lines for `mnist_plot.plt` as well?

Reply:
OK. Fixed and amended.
It should be fixed correctly.

https://github.com/Alignof/linfa/commit/69ef8cb78973a47e0c6300725f6392b92cd0e5fa#diff-69a53e0c2347812bff90dd11e1ac389f7fc9793ac0b45deee06bd17f8186be05